int8 ec_id          # ethercat板子的索引
int8 slave_id       # ethercat板子的子板的索引
int8 passage        # 通道号
int8 motor_id_old   # 旧电机id
int8 motor_id_new   # 新电机id